<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACAudioCycleOsgQt</class>
 <widget class="QMainWindow" name="ACAudioCycleOsgQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>724</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AudioCycleOsgQt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupControls">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>225</width>
        <height>600</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Controls</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QGroupBox" name="groupBoxBrowserControls">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>201</width>
         <height>381</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Browser Controls</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QGroupBox" name="groupBoxSimilarity">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>181</width>
          <height>85</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Similarity</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QCheckBox" name="checkBoxRhythm">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>87</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Rhythm</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxTimbre">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>87</width>
           <height>20</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Timbre</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBoxHarmony">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>87</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Harmony</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBoxClusters">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>181</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>Clusters Mode</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QSlider" name="sliderClusters">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
        <widget class="QLabel" name="labelClustersNumber">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>54</width>
           <height>18</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Number</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxClustersMethod">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>60</y>
           <width>111</width>
           <height>26</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>KMeans (default)</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="labelClustersMethod">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>54</width>
           <height>18</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Clustering</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelClustersPositions">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>54</width>
           <height>18</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Positions:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxClustersPositions">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>80</y>
           <width>111</width>
           <height>26</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Propeller (default)</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButtonRecenter">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>350</y>
          <width>80</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Recenter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonClean">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>320</y>
          <width>80</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Clean</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonLaunch">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>80</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Launch</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBoxNeighbors">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>181</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Neighbors Mode</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QLabel" name="labelNeighborsMethod">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>54</width>
           <height>18</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Neighbors</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelNeighborsPositions">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>54</width>
           <height>18</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Positions:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxNeighborsPositions">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>111</width>
           <height>26</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>None available</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="comboBoxNeighborsMethod">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>111</width>
           <height>26</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>None available</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButtonBack">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>51</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonForward">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>350</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonClusters">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>280</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clusters</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonNeighbors">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>300</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Neighbors</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelChosenMode">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>71</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Chosen mode:</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupAudioControls">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>530</y>
         <width>201</width>
         <height>111</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Audio Controls</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QLabel" name="labelAudioBPM">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>54</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>BPM</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelAudioPitch">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>54</width>
          <height>18</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
       <widget class="QSlider" name="sliderBPM">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>220</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
       <widget class="QSlider" name="sliderPitch">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>25</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>50</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonMuteAll">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string>Mute All</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBoxOsc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>131</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>OSC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QGroupBox" name="groupBoxOscControl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Receive control from:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <widget class="QLineEdit" name="lineEditControlIP">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="IP"/>
         </property>
         <property name="text">
          <string>localhost</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditControlPort">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>12345</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonControlStart">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBoxOscFeedback">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>Send feedback to:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QLineEdit" name="lineEditFeedbackIP">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>localhost</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEditFeedbackPort">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>12346</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonFeedbackStart">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>51</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBrowser">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="title">
       <string>Browser</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="ACOsgBrowserViewQT" name="browserOsgView" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Media_Directory"/>
    <addaction name="actionLoad_Media_Files"/>
    <addaction name="actionLoad_ACL"/>
    <addaction name="separator"/>
    <addaction name="actionSave_ACL"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Media_Directory">
   <property name="text">
    <string>Load Media Directory</string>
   </property>
  </action>
  <action name="actionLoad_ACL">
   <property name="text">
    <string>Load ACL</string>
   </property>
  </action>
  <action name="actionSave_ACL">
   <property name="text">
    <string>Save ACL</string>
   </property>
  </action>
  <action name="actionLoad_Media_Files">
   <property name="text">
    <string>Load Media Files</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ACOsgBrowserViewQT</class>
   <extends>QWidget</extends>
   <header>ACOsgBrowserViewQT.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
